src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -201,6 +201,17 @@
                 namespace,
                 (state: ADRState) => state.adrError
             ),
+            adrErrorText() {
+                let errorKey = "errorFetchingDatasetsFromADR";
+                // When we get an error matching this this is because a user has tried to pull data from the ADR
+                // after authenticating with Auth0 before they have logged into the ADR at all.
+                // In this scenario they do not have an ADR account yet, so need to login there first for the
+                // fetch datasets to work.
+                if (this.adrError && this.adrError.detail && this.adrError.detail.match("has not yet logged into ADR")) {
```

Comment:
```suggestion
                if (this.adrError?.detail?.match("has not yet logged into ADR")) {
```
should also work, the ? will just return undefined if the property does not exist and not continue down the chain